ext/data/recommended-dictionaries.json

```diff
@@ -0,0 +1,15 @@
+{
+    "ja": {
+        "terms": [
+            {"name": "Jitendex", "url": "https://github.com/stephenmk/stephenmk.github.io/releases/latest/download/jitendex-yomitan.zip"}
+        ],
+        "kanji": [
+            {"name": "Kanjidic English", "url": "https://github.com/themoeway/jmdict-yomitan/releases/latest/download/KANJIDIC_english.zip"}
+        ],
+        "frequency": [
+            {"name": "BCCWJ-LUW", "url": "https://github.com/toasted-nutbread/yomichan-bccwj-frequency-dictionary/releases/download/1.0.1/BCCWJ-LUW.zip"}
+        ],
+        "grammar": [],
+        "pronunciation": []
+    }
+}
```

Comment:
Now that updating is in, would it take a lot of work to list `index.json` files instead?

Reply:
Would have to ask @StefanVukovic99 on that one. But I think it's probably better to let another pr take that one. And static links will still need to be supported even if we add index.json support.

Reply:
Yeah, I'd like that so we can show more info, like the description, but not necessary for now